Makefile

```diff
@@ -11,6 +11,12 @@ PKGS = $(shell go list ./... | grep -v /vendor/)
 gofmtcheck:
 	@go fmt $(PKGS) | grep ".*\.go"; if [ "$$?" = "0" ]; then exit 1; fi     
 
+doccheck:
+	grep -hoE '(kube_[^ |]+)' Documentation/* | sort -u > documented_metrics
```

Comment:
To that end, maybe don't add this as a comment but as output before the check is run.

```
@echo "Checking if documentation is up to date..."
@echo "Metrics with - are present in tests but missing in documentation, metrics with + are documented but not tested."
```

(too verbose?)